package.json

```diff
@@ -35,8 +35,13 @@
     "is-url": "^1.2.4",
     "jayson": "^2.1.2",
     "jest": "^24.5.0",
+    "json-schema-faker": "^0.5.0-rc16",
```

Comment:
remove this - I was gonna use this to try to use a different tool than quicktype that would generate types from json examples, not json schema.